docs/directory_setup.rst

```diff
@@ -0,0 +1,21 @@
+Setting Up a Template Directory
```

Comment:
What about making this page more generic - "abc-classroom quickstart" - and describing all of the steps to get going with the a new course?

Reply:
excellent idea!!

Reply:
@nkorinek i love this idea.. let's do it!